### ISSUE TITLE
✨ Overall dockwidget UI feedback

### ISSUE THREAD

**Issue by goanpeca** [napari/napari#3103]
## 🚀 Feature

Plugins have very long names and when tabbed together are not very readable (if more than… 2 or 3).

<img width="224" alt="Screen Shot 2021-07-29 at 10 00 21 AM" src="https://user-images.githubusercontent.com/3627835/127515782-a7292b0d-5a2d-4a82-9151-c91a48d63e4a.png">

<img width="589" alt="Screen Shot 2021-07-29 at 10 00 37 AM" src="https://user-images.githubusercontent.com/3627835/127515830-b6ebb760-9d0f-47b2-8769-0498a1ffa629.png">

- Buttons on top are very small
- Text contrast is not suficient
- movable splitters between dock areas are hard to grab.
- dock widgets can be tricky to re-dock.

---

Pinging @liaprins-czi

**Comment by liaprins-czi** [comment]
Very related to this heuristic issue (possible duplicate on most fronts):
https://github.com/napari/product-heuristics-2020/issues/18

If useful we can still kepp this one open though

**Comment by liaprins-czi** [comment]
### Overview of design need
- **Is there an existing GitHub issue this design work pertains to?** 
[Layer panels layout #18](napari/product-heuristics-2020#18)

- **In a sentence or two, describe the problem to be solved for users**
Very confusing interactions with the visibility + decoupling of the layer controls and layer list panels: The icon to click on to pop them out is very tiny and hard to click. This icon is also not very clear in communicating what will happen if click it. More importantly, when I try to drag popped-out panels back into the main viewer, they don't go back to the way they started: now they are "overlapped" and can only get to them by the tabs (also unexpected to have tabs at the bottom, usually tabs are at the top).

### What level of design is needed? (Choose all that apply)
- [X] **User experience research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
- [X] **Information flow / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
- [X] **Visual:** creating mockups, icons, etc
(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)

### Is design a blocker?
- [X] **Yes:** engineering cannot proceed without a design first
- [ ] **No:** engineering can create a first version, and design can come in later to iterate and refine

If selecting **Yes**, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?
Development of this issue is dependent upon design, however, there is no need for dev to work on this until it is ready.

### T-shirt size
- [ ] ●○○○○ XS: able to be done in a typical workday or two
- [ ] ●●○○○ S: able to do within one sprint, if also working on other things in the sprint
- [ ] ●●●○○ M: able to do within one sprint, if it is the sole focus
- [X] ●●●●○ L: will likely take more than one sprint to complete, but less than two
- [ ] ●●●●● XL: a very large experience to be determined or overhauled, will need to be broken down further

**Comment by tlambert03** [comment]
> The icon to click on to pop them out is very tiny and hard to click. 

agree... but difficult to make bigger without increasing the size of the title bar itself, which I wouldn't want to get much bigger (it's mostly dead space). 

> This icon is also not very clear in communicating what will happen if click it.

<img width="33" alt="Untitled" src="https://user-images.githubusercontent.com/1609449/130528552-3086e2bc-76c3-4121-94ba-046302a8331b.png">

I don't know :) I think it says "window on top of window".  But always nice to have better ones!

> More importantly, when I try to drag popped-out panels back into the main viewer, they don't go back to the way they started: now they are "overlapped" and can only get to them by the tabs 

this depends on how you drop them (that flexibility is nice if you have more than a single widget, which can easily happen with multiple plugins open)

https://user-images.githubusercontent.com/1609449/130528361-9b59b108-3cfc-4053-bb0e-4b14c0050e5d.mov

> (also unexpected to have tabs at the bottom, usually tabs are at the top).

agreed.  easy to change with https://doc.qt.io/qt-5/qmainwindow.html#setTabPosition

**Comment by willingc** [comment]
Bumping this one re: usability and accessibility

As someone with aging eyes, it is very difficult to see the controls for individual panels. This screenshot makes the eye and copy? look larger than they appear on a smaller screen.

<img width="339" height="182" alt="Image" src="https://github.com/user-attachments/assets/36669b79-fb21-4e19-a6ce-2b36a8113deb" />

**Comment by willingc** [comment]
Related to: #7471

**Comment by psobolewskiPhD** [comment]
Agreed. We should allow UI scaling, see #8820

**Comment by psobolewskiPhD** [comment]
In https://github.com/napari/napari/pull/8714 the sizes and contrast were increased.
The font size has also been bumped: https://github.com/napari/napari/pull/8948
Re-docking can be accomplished by using the button.
Current view of the elements from the OP:

<img width="358" height="346" alt="Image" src="https://github.com/user-attachments/assets/0f2d473d-b054-4776-a939-5eaadf4cdd1d" />

I do agree the splitter is small.
